doc/source/bindings.rst

```diff
@@ -28,8 +28,9 @@ TARGET:
 | **Gettable**: yes
 | **Settable**: yes
 | **Type**: `Vessel <structures/vessels/vessel.html>`__ or `Body <structures/celestial_bodies/body.html>`__
-| **Description**: Whichever `Orbitable <structures/orbits/orbitable.html>`__ object happens to be the one selected as the current KSP target. If set to a string, it will assume the string is the name of a vessel being targeted and set it to a vessel by that name. For best results set it to Body("some name") or Vessel("some name") explicitly.  This will throw an exception if called from a vessel other than the active vessel, as limitations in how KSP sets the
-target vessel limit the implementation to working with only the active vessel.
+| **Description**: Whichever `Orbitable <structures/orbits/orbitable.html>`__ object happens to be the one selected as the current KSP target. If set to a string, it will assume the string is the name of a vessel being targeted and set it to a vessel by that name. For best results set it to Body("some name") or Vessel("some name") explicitly.  This will throw an exception if called from a vessel other than the active vessel, as limitations in how KSP sets the target vessel limit the implementation to working with only the active vessel.  
+
+.. _hastarget:
```

Comment:
I think I'm just going to reformat these as proper bullet lists.  I don't get what these originally were supposed to be doing and it was that way before I touched it.